external-crates/move/documentation/book/src/modules.md

```diff
@@ -18,8 +18,8 @@ module <address>::<identifier> {
 }
 ```
 
-where `<address>` is a valid [named or literal address](./address.md) specifying the module's
-package.
+where `<address>` is a valid [named or literal address](./primitive-types/address.md) specifying the
+module's package.
```

Comment:
Consider removing the discussion around explicit numerical addresses when possible, to discourage usage?

```suggestion
where `<address>` is a valid [address](./primitive-types/address.md) specifying the
module's package.
```

And then possibly add a section at the end that mentions numerical addresses and that they can be useful for testing? (Or... can they be, for actually testing sui code?)